### ISSUE TITLE
Merge flag for updateconf doesnt update existing CONTEXT attributes

### ISSUE THREAD

**Issue by OpenNebulaSupport** [OpenNebula/one#5716]
**Description**
When setting `update_type = 1` for `one.vm.updateconf` XML-RPC API call, instead of merging the new attribute (as stated in the docs):

```
Update type: 0: Replace the whole template. 1: Merge new template with the existing one.
```  

it behaves like the `append` operation. It just adds new attributes without modifying existing ones.

**To Reproduce**
Try to update an existing `CONTEXT` attribute with  `update_type = 1`.

**Expected behavior**
Existing attributes are overwritten when `update_type = 1`.

**Details**
 - Affected Component: oned
 - Version: 6.2

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by paczerny** [comment]
PRs:
* [src](https://github.com/OpenNebula/one-ee/pull/1728)
* [tests](https://github.com/OpenNebula/development/pull/1726)
* [docs](https://github.com/OpenNebula/docs/pull/2125)

**Comment by paczerny** [comment]
Fix encrypted attributes after updateconf, PRs:
* [src](https://github.com/OpenNebula/one-ee/pull/1741)
* [tests](https://github.com/OpenNebula/development/pull/1732)

**Comment by paczerny** [comment]
Fixed in 6.2.1
